File: src/Entity/News.php
<?php

namespace App\Entity;
use Symfony\Component\Validator\Constraints as Assert;

use App\Repository\NewsRepository;
use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\Common\Collections\Collection;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Serializer\Annotation\Groups;

/**
 * @ORM\Entity(repositoryClass=NewsRepository::class)
 * @ORM\HasLifecycleCallbacks()
 * @ORM\Table(name="News")
 */
class News
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     * @ORM\Column(type="integer")
     * @Groups("post:read")
     */
    private $id;

    /**
     * @ORM\Column(type="string")
     *  @Assert\Length(
     *      min = 2,
     *      max = 11,
     *      minMessage = "Your first name must be at least {{ limit }} characters long",
     *      maxMessage = "Your first name cannot be longer than {{ limit }} characters"
     * )
     * @Assert\NotNull
     * @Groups("post:read")
     */
    private $title;

    /**
     * @ORM\Column(type="string", length=500, nullable=true)
     * @Assert\File(maxSize="500k", mimeTypes={"image/jpeg", "image/jpg", "image/png", "image/GIF"})
     *@Assert\NotNull
     *@Groups("post:read")
     */
    private $bg_img;

    /**
     * @ORM\Column(type="string", length=500)
     * @Assert\File(maxSize="500k", mimeTypes={"image/jpeg", "image/jpg", "image/png", "image/GIF"})
     * @Groups("post:read")
     */
    private $img;

    /**
     * @ORM\Column(type="string")
     * @Assert\Length(
     *      min = 2,
     *      max = 50,
     *      minMessage = "Your first name must be at least {{ limit }} characters long",
     *      maxMessage = "Your first name cannot be longer than {{ limit }} characters"
     *
     * )
     * @Assert\NotNull
     * @Groups("post:read")
     */
    private $description;

    /**
     *@var \DateTime
     * @ORM\Column(type="date",name="creation_date")
     * @Groups("post:read")
     */
    private $creation_date;


    public function __construct()
    {
        $this->comments = new ArrayCollection();
    }

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getTitle(): ?string
    {
        return $this->title;
    }

    public function setTitle(string $title): self
    {
        $this->title = $title;

        return $this;
    }

    public function getBgImg(): ?string
    {
        return $this->bg_img;
    }

    public function setBgImg(string $bg_img): self
    {
        $this->bg_img = $bg_img;

        return $this;
    }

    public function getImg(): ?string
    {
        return $this->img;
    }

    public function setImg(string $img): self
    {
        $this->img = $img;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }
    /**
     * @return \DateTime creation_date
     */
    public function getCreationDate(): ?\DateTimeInterface
    {
        return $this->creation_date;
    }
    /**
     * Set creation_date
     *
     * @param \DateTime $creation_date
     *
     * @return MisesEnPlace
     */
    public function setCreationDate(\DateTimeInterface $creation_date): self
    {
        $this->creation_date = $creation_date;

        return $this;
    }



}

File: src/com/mycompany/myapp/entities/News.java

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

import java.util.Date;

/**
 *
 * @author Lenovo
 */
public class News {
    //nemchio taw nchofo entity fi symfony
    
      private int id;

    public News(int id, String title) {
        this.id = id;
        this.title = title;
    }

    public News(int id, String title, String bg_img, String img, String description) {
        this.id = id;
        this.title = title;
        this.bg_img = bg_img;
        this.img = img;
        this.description = description;
    }
    private String title;
    private String bg_img;
    private String img;
    private String creation_date;
    private String description;

    public News(String title, String bg_img, String img, String description) {
        this.title = title;
        this.bg_img = bg_img;
        this.img = img;
        this.description = description;
    }

    public News() {
    }

    public int getId() {
        return id;
    }

    public News(int id, String title, String bg_img, String img, String creation_date, String description) {
        this.id = id;
        this.title = title;
        this.bg_img = bg_img;
        this.img = img;
        this.creation_date = creation_date;
        this.description = description;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getBg_img() {
        return bg_img;
    }

    public void setBg_img(String bg_img) {
        this.bg_img = bg_img;
    }

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }

    public String getCreation_date() {
        return creation_date;
    }

    public void setCreation_date(String creation_date) {
        this.creation_date = creation_date;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    @Override
    public String toString() {
        return "News{" + "id=" + id + ", title=" + title + ", bg_img=" + bg_img + ", img=" + img + ", creation_date=" + creation_date + ", description=" + description + '}';
    }

    
    
    
   
    
    
    
    
    }
